mongo/options/clientencryptionoptions.go

```diff
@@ -6,10 +6,17 @@
 
 package options
 
+import (
+	"crypto/tls"
+	"errors"
+	"fmt"
+)
+
 // ClientEncryptionOptions represents all possible options used to configure a ClientEncryption instance.
 type ClientEncryptionOptions struct {
 	KeyVaultNamespace string
 	KmsProviders      map[string]map[string]interface{}
+	TLSConfig         map[string]tls.Config
```

Comment:
A `tls.Config` contains a `sync.RWMutex` and must not be copied. Change to `map[string]*tls.Config` to prevent copying the values.

Reply:
Ah good to know! Fixed now.